client/src/pages/boxes/SettleBoxPage4/SettleBoxPage4.tsx

```diff
@@ -0,0 +1,186 @@
+import { Item } from 'rc-menu';
+import s from '../BoxesPage.module.less';
+import { Button } from 'antd';
+
+interface BoxData {
+  volunteerId: string;
+  boxId: string;
+  plnAmount: Record<string, number>;
```

Comment:
Doesn't fit further data definition - plnAmount should be array of interfaces (makes more sense than using Record in this scenario). And it would be a good idea to replace string type with e.g. `type denomination = '1gr' | '2gr' | '5gr' ...`